packages/sources/tradingeconomics-test/src/endpoint/price.ts

```diff
@@ -0,0 +1,63 @@
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { EndpointTypes } from './price-router'
+import { PriceEndpointParams } from '@chainlink/external-adapter-framework/adapter'
+const logger = makeLogger('TradinEconomics HTTP')
+
+const transformInput = (input: PriceEndpointParams) => {
+  const base = input.base.replace(' ', '').split(',')
+  const quote = input.quote.replace(' ', '').split(',')
+  return base
+    .map((element, index) => {
+      return `${element}:${quote[index]}`
+    })
+    .join(',')
+}
```

Comment:
Where is this coming from?

Reply:
This is because the endpoint accepts batch requests and in v2 they don't handle it that way. 
https://docs.tradingeconomics.com/?javascript#snapshot-markets10

example request:
https://api.tradingeconomics.com/markets/symbol/aapl:us,gac:com?c=guest:guest&f=json

Reply:
This seems a bit different than how v2 is doing this though. In v2, only the `base` param is being used - while this uses both the `base` and `quote`, which seems like it would cause different outputs?

Reply:
I think I have the same concern here. I don't think `params` in `prepareRequest` would be the format that you're parsing here. It would just a list of individual requests sent to the adapter with overridden symbols. So something like `[{"base":"ETHUSD:CUR", "quote":"USD"},{"base":"BTCUSD:CUR", "quote":"USD"}]`. Which is why I think v2 uses just `base` as the symbol for these requests.